single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/tests/test_synapse_pipeline_ingest_melbparkingdata.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for Synapse pipeline: process_covid_results"""
+
+import os
+from dataconnectors import adls, local_file
+from utils import pipelineutils
+
+PIPELINE_NAME = os.getenv("PIPELINE_NAME")
+TRIGGER_NAME = os.getenv("TRIGGER_NAME")
+
+
+def test_synapse_pipeline_succeeded(synapse_client, sql_connection, adls_connection_client):
+
+    # Upload file
+    container = os.getenv("ADLS_DESTINATION_CONTAINER")
+    filepath = os.getenv("SOURCE_FILE_PATH")
+    filename = os.getenv("SOURCE_FILE_NAME")
+    processed_container = os.getenv("ADLS_PROCESSED_CONTAINER")
+
+    request_id = adls.upload_to_ADLS(adls_connection_client, container, filepath, filename, "")
+    # pipeline_run_id = pipelineutils.get_pipeline_by_request_id(synapse_client, request_id, PIPELINE_NAME, TRIGGER_NAME)
+    pipeline_run_id="5975d7b9-f089-4c36-a35a-02a0684a5a4d"
```

Comment:
Need to fix this, it's only looking at one pipeline run

Reply:
Fixed. GREAT catch :)